flake.nix

```diff
@@ -20,29 +20,15 @@
         flake-utils.follows = "flake-utils";
         nix-3rdparty.follows = "nix-3rdparty";
         nil-crypto3.follows = "nil-crypto3";
-        nil-zkllvm-blueprint.follows = "nil-zkllvm-blueprint";
       };
     };
     nil-crypto3 = {
-      type = "github";
-      owner = "NilFoundation";
-      repo = "crypto3";
-      rev = "3bd5b8df2091274abaa28fd86b9e3e89d661b95a";
-      inputs = {
-        nixpkgs.follows = "nixpkgs";
-        flake-utils.follows = "flake-utils";
-        nix-3rdparty.follows = "nix-3rdparty";
-      };
-    };
-    nil-zkllvm-blueprint = {
-      url = "https://github.com/NilFoundation/zkllvm-blueprint";
+      url = "https://github.com/NilFoundation/crypto3";
```

Comment:
what reason switch to git type

Reply:
It's just a copy-paste from zkllvm. Do you have a strong preference?